ext/AvizExt/viz/sensitivity.jl

```diff
@@ -1,6 +1,18 @@
 using Statistics
 using Printf
 using ADRIA.sensitivity: _get_cat_quantile
+using ADRIA: _is_discrete_factor
+
+"""
+    _get_guided_labels()::Vector{String}
+
+    Returns labels for categories of the `guided` factor.
```

Comment:
No indentation for the description.

```suggestion
Returns labels for categories of the `guided` factor.
```